<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>1124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;360 Light field display simulator&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Carlo Nicolini carlo.nicolini@iit.it&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Feb 2013&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxProjector">
         <property name="title">
          <string>Projector</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelProjCenterX">
            <property name="text">
             <string>ProjCenterX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxProjCenterX"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelProjCenterY">
            <property name="text">
             <string>ProjCenterY</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxProjCenterY"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelProjCenterY_2">
            <property name="text">
             <string>ProjCenterZ</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxProjCenterZ"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelProjPitch_2">
            <property name="text">
             <string>ProjPitch</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxProjPitch"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelProjRoll">
            <property name="text">
             <string>ProjRoll</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxProjRoll"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelProjFocal">
            <property name="text">
             <string>ProjFocal</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxProjFocal"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelProjAperture">
            <property name="text">
             <string>ProjAperture</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxProjAperture"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelProjAreaEdge">
            <property name="text">
             <string>ProjEdge</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxProjAreaEdge"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxViewer">
         <property name="title">
          <string>Viewer</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelViewerDistance">
            <property name="text">
             <string>Distance</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxViewerDistance"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelViewerHeight">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxViewerHeight"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxMirror">
         <property name="title">
          <string>Mirror</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelMirrorAngle">
            <property name="text">
             <string>Angle</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxMirrorAngle"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Freq [cyc/s]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxMirrorFreq"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxOptions">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkboxAutorotation">
            <property name="text">
             <string>Autorotation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkboxViewRaysForward">
            <property name="text">
             <string>ViewRays forward</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxViewRaysBackward">
            <property name="text">
             <string>View rays backward</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxRayIndex"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Data</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="labelProjPlaneCoords">
              <property name="text">
               <string>ModelViewMatrix</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEditModelViewMatrix">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="labelMVPmatrix">
              <property name="text">
               <string>ModelViewProjection Matrix</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEditModelviewProj">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="GLWidgetGeometry" name="glWidgetGeometry" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="GLWidgetShader" name="glShaderWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_mesh_file_obj"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_mesh_file_obj">
   <property name="text">
    <string>Open mesh file (.obj)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidgetGeometry</class>
   <extends>QWidget</extends>
   <header>GLWidgetGeometry.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLWidgetShader</class>
   <extends>QWidget</extends>
   <header>GLWidgetShader.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
